src/mixins/object_interactivity.mixin.js

```diff
@@ -311,5 +311,13 @@
       return false;
     },
 
+    renderDragSourceEffect: function() {
+      // for subclasses
+    },
+
+    renderDropTargetEffect: function(/* e, dontClear */) {
+      // for subclasses
+    },
+
```

Comment:
this make clear that if you want to support drag/drop you have to define at least those 3 ( i will try to move the event handlers on Object interactivity ( another week )